src/angular/components/set-password.component.ts

```diff
@@ -21,8 +21,10 @@ import { SetPasswordRequest } from '../../models/request/setPasswordRequest';
 
 import { ChangePasswordComponent as BaseChangePasswordComponent } from './change-password.component';
 
+import { Directive } from '@angular/core';
```

Comment:
Did this pass lint check? Does `@` come before `.` in alphabetical/lexicographical sort order? Perhaps these should be separate import groups?